### PR TITLE
Add latest MEI related publications

### PR DESCRIPTION
**Pull request by musicEnfanthen**
This PR adds two of the latest MEI related publications.

### CODE REVIEW

**Review by rettinghaus — APPROVED** [review]

File: resources/mei_bibliography.bib [hunk 1]
```diff
@@ -587,6 +597,21 @@ @misc{Morent_2006
...
+ author = {Münnich, Stefan},
+ title = {Handwritten -- Encoded -- Semantically Operationalised: Musical Writing Scenes in the Digital Realm},
+ pages = {371--391},
+ publisher = {{Brill | Fink}},
+ isbn = {9783770567140},
+ series = {Theorie der musikalischen Schrift},
```
Comment (doerners) [inline resources/mei_bibliography.bib:607]: the volume seems to be missing, I would add that

**Review by doerners — COMMENTED** [review]
I would add the volume for the incollection item, otherwise the changes seem fine to me and could be merged

File: resources/mei_bibliography.bib (hunk 1 above)
Comment (musicEnfanthen) [inline resources/mei_bibliography.bib:607]:
```suggestion
 series = {Theorie der musikalischen Schrift},
 volume = {4},
```

Comment (musicEnfanthen) [reply resources/mei_bibliography.bib:607]: Good catch, thanks! Added the missing volume

**Review by doerners — APPROVED** [review]

**Review by rettinghaus — APPROVED** [review]

**Pull request merged**